### PR TITLE
Math magician Api

### PR DESCRIPTION
**Pull request by aster-alemu**
In this PR, I have been doing the following activities.

1. Use the following external API: [API Ninjas](https://api-ninjas.com/api/quotes)
2. Create a new component for displaying a quote
3. Fetch data inside of a new component
4. Show a loading state
5. Show an error state
6. Import the new component into the main component (next to the Calculator)

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @aster-alemu,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### These should be header links as we have in the template. Kindly fix your readme to follow the readme template.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -15,7 +14,7 @@
   🙏 Acknowledgements
   📝 License
```
Comment (tugsem) [inline README.md:15]: - [ ] Please use the correct format for listing the menu in your readme file. You can copy the template which is written in md format. It should look like this: https://github.com/microverseinc/readme-template.

Your readme looks like this: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,3 @@
-# Math magicians: events
 
 📗 Table of Contents
   📖 About the Project
```
Comment (tugsem) [inline README.md]: - [ ] Please use the correct format for listing the menu in your readme file. You can copy the template which is written in md syntax. It should look like this: https://github.com/microverseinc/readme-template.

Your readme looks like this: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @aster-alemu 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️quote rendered on the same page as the calculator
✔️API used correctly
✔️useState and props used correctly
## Required Changes ♻️
- [ ] Please add the right links for your GitHub and Twitter accounts as the LinkedIn account. Right now, they are not clickable.
image

- [ ] Please add a license to your project using this link and change it to your name. https://choosealicense.com/licenses/mit/

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -91,6 +88,6 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
 -Inspiration etc
 
-📝 License
+# 📝 License
 
-This project is not licensed.
```
Comment (IngDavidTM) [inline README.md:96]: - [ ] While you did a fantastic job, as requested before, please link the license you created to the README file 👍

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @aster-alemu 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @aster-alemu, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**